libs/common/src/constants/app.constants.ts

```diff
@@ -3,3 +3,9 @@ export const SUCCESS_MESSAGES = {
   USER_LOGGED_OUT: 'User logged out successfully',
   USER_LOGGED_IN: 'User logged in successfully',
 };
+
+export const ERROR_MESSAGES = {
+  USER_NOT_FOUND: 'User not found',
+  USER_ALREADY_EXISTS: 'User already exists',
+  UNIQUE_CONSTRAINT_VIOLATION: 'duplicate key value violates unique constraint',
+};
```

Comment:
Try avoiding this approach and instead type the error where you throw it. It's really hard to track where this is error is coming from